### PR TITLE
Created the EventCreate component

### PR DESCRIPTION
**Pull request by Markkop**
Products were removed and replaced by Events
Form created with Formik, but without yup for now
Created Mutation root type in schema.js

### CODE REVIEW

File: packages/server/graphql/schema.js [hunk 1]
```diff
@@ -3,31 +3,75 @@ const {
...
+      type: GraphQLList(eventGraphQLType),
       resolve() {
-        return Product.find().lean();
+        return Event.find();
       }
     }
   }
 });
 
+const EventCreate = mutationWithClientMutationId({
```
Comment (jgcmarins) [inline packages/server/graphql/schema.js]: move this to another file

File: packages/server/graphql/schema.js [hunk 2]
```diff
@@ -3,31 +3,75 @@ const {
...
       args: { id: { type: GraphQLNonNull(GraphQLID) } },
       resolve(parent, args) {
-        return Product.findById(fromGlobalId(args.id).id);
+        return Event.findById(fromGlobalId(args.id).id);
       }
     },
-    products: {
-      type: GraphQLList(productGraphQLType),
+    events: {
+      type: GraphQLList(eventGraphQLType),
```
Comment (jgcmarins) [inline packages/server/graphql/schema.js]: this is good, but won't scale
we should use connection for pagination:
https://github.com/facebook/relay/blob/a1e9a4b395f3eed6dfe58703a5fd02d0765b0672/packages/relay-runtime/handlers/connection/RelayConnectionInterface.js#L19-L28

**Comment by jgcmarins** [comment]
good job, keep going

File: packages/server/graphql/schema.js (hunk 1 above)
Comment (Markkop) [reply packages/server/graphql/schema.js]: just did with the new subscription branch

File: packages/server/graphql/schema.js (hunk 2 above)
Comment (Markkop) [reply packages/server/graphql/schema.js]: I'll add pagination by the end of the project. It seems a little complex x_x

**Pull request merged**